frontend/src/components/forms/CreateLog.tsx

```diff
@@ -273,9 +325,9 @@ const CreateLog = () => {
                   <FormControl isRequired>
                     <FormLabel>Employee</FormLabel>
                     <Select
-                      options={EMPLOYEES}
+                      options={employeeOptions.length > 0 ? employeeOptions : []}
                       isDisabled
-                      defaultValue={{ label: employee, value: employee }} // needs to be the current user
+                      defaultValue={{ label: employee.label, value: employee.value }} // needs to be the current user
```

Comment:
Going off my comment above, could this call a function that just gets the current user?